lang/java/avro/src/test/java/org/apache/avro/io/parsing/RecursiveSymbolTest.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Instead of adding a new class, could you please add a new test method with a meaningful name to TestSchemaValidation?